converters/toZigbee.js

```diff
@@ -2543,6 +2543,94 @@ const converters = {
             sendTuyaCommand(entity, 528, 0, [4, 0, 0, ...payloadValue]);
         },
     },
+    etop_thermostat_system_mode: {
+        key: ['system_mode'],
+        convertSet: async (entity, key, value, meta) => {
+            switch (value) {
+            case 'off':
+                await sendTuyaCommand(entity, 257, 0, [1, 0/* off */]);
+                break;
+            case 'heat':
+                await sendTuyaCommand(entity, 257, 0, [1, 1/* on */]);
+                await utils.sleepMs(500);
+                await sendTuyaCommand(entity, 1028, 0, [1, 0/* manual */]);
+                break;
+            case 'auto':
+                await sendTuyaCommand(entity, 257, 0, [1, 1/* on */]);
+                await utils.sleepMs(500);
+                await sendTuyaCommand(entity, 1028, 0, [1, 2/* auto */]);
+                break;
+            }
+        },
+    },
+    etop_thermostat_away: {
+        key: ['away'],
```

Comment:
I think it's better to change this to `away_mode`, seems Home Assistant also uses this.